packages/x-date-pickers/src/DesktopDatePicker/DesktopDatePicker.test.tsx

```diff
@@ -46,6 +46,32 @@ describe('<DesktopDatePicker />', () => {
     expect(getByTestId('component-test')).not.to.equal(null);
   });
 
+  it('prop: components.PaperContent', () => {
+    function CustomPaperContent({ children }) {
+      return (
+        <div>
+          <p>test custom content</p>
+          {children}
+        </div>
+      );
+    }
+    render(
+      <DesktopDatePicker
+        renderInput={(params) => <TextField {...params} />}
+        onChange={() => {}}
+        value={null}
+        components={{
+          PaperContent: CustomPaperContent,
+        }}
+      />,
+    );
+
+    openPicker({ type: 'date', variant: 'desktop' });
+
+    expect(screen.getByText('test custom content')).not.equal(null);
+    expect(screen.getByText('January 1970')).not.equal(null);
```

Comment:
This test is failing on Edge: https://app.circleci.com/pipelines/github/mui/mui-x/22602/workflows/82b0a275-340e-426d-88e6-337221d41e49/jobs/130351

Using `screen.getByText` is sometimes problematic because Edge has a different timezone, on BrowserStack, than other browsers. Is there a way to use another text that doesn't contain the full month + year? In a different timezone it might not be January. If not, then you need to first convert the date to a string so it can be asserted.

We had a similar problem in the past about this. Maybe https://github.com/mui/mui-x/pull/5221 helps.

Reply:
Thanks for pointing this out.
Have we considered ["locking" in the Timezone](https://blog.liprandi.net/2020/10/running-karma-in-specific-timezone.html) to have consistent results across the board?
Or do we want this dependency on various time zones to catch (un)expected issues?

Reply:
Created [a PR](https://github.com/mui/mui-x/pull/5955) to fix it.